meson_options.txt

```diff
@@ -1,4 +1,4 @@
-option('tests', type : 'boolean', value : true, description : 'enable tests')
+option('tests', type : 'boolean', value : false, description : 'enable tests')
```

Comment:
I still find this quite sad and feel like we ought to be able to do better. But the upstream `.gitlab-ci/run-tests.sh` does the following which also feels like a lot:

```bash
# FIXME: The tests should be isolated and use mock services so they do not
# require a functioning system bus. This will have to do for now though.
sudo mkdir -p /run/dbus
sudo mkdir -p /var
sudo ln -s /var/run /run
sudo dbus-daemon --system --fork
#sudo /usr/lib/polkit-1/polkitd --no-debug &
#sudo /usr/libexec/fwupd/fwupd --verbose &

# FIXME: Running the flatpak tests as root means the system helper doesn’t
# need to be used, which makes them run a lot faster.
sudo \
dbus-run-session -- \
meson test \
        -C _build \
        --timeout-multiplier ${MESON_TEST_TIMEOUT_MULTIPLIER} \
        --no-suite flaky \
        --print-errorlogs \
        "$@"
```